documentation/modules/security/proc-installing-your-own-ca-certificates.adoc

```diff
@@ -22,49 +32,93 @@ The chain should be in the following order:
 2. One or more intermediate CAs
 3. The root CA
 +
-** All CAs in the chain should be configured as a CA in the X509v3 Basic Constraints.
+** All CAs in the chain should be configured using the X509v3 Basic Constraints extension. Basic Constraints limit the path length of a certificate chain.
+* The OpenSSL TLS management tool for converting certificates.
+
+.Before you begin
+The Cluster Operator generates the following files for the `_CLUSTER-NAME_-cluster-ca-cert` secret:
+
+* `ca.crt` cluster certificate in PEM format
+* `ca.p12` cluster certificate in PKCS #12 format
+* `ca.password` to access the PKCS #12 file
+
+If you are using a service registry, make sure you include all three files in your secret.
+The service registry can then use a certificate in the appropriate format to connect securely with Kafka.
```

Comment:
I don't think this should be here. Not all service registries require PKCS12 files and not all applications which need PKCS12 files are service registries. Maybe we can use something like this?

```suggestion
Some applications cannot use the PEM certificates directly and support only the PKCS #12 format.
To make it easier to use them, you can also add the cluster certificate in PKCS #12 format.
```

Reply:
Better. I thought I needed to say something specifically about service registries. 
Changed to:

> Some applications cannot use PEM certificates and support only PKCS #12 certificates. You can also add your own cluster certificate in PKCS #12 format.